build-system/tasks/issues-tasks.js

```diff
@@ -0,0 +1,255 @@
+var BBPromise = require('bluebird');
```

Comment:
@erwinmombay Why don't we run lint against these files?

Reply:
it was because we wrote in es5 style with our build tools since we use node 4 as baseline for our tools. we could either upgrade or have separate lint rules

Reply:
I opened #11273 so we add separate lint rules for the build system. Let me know if I didn't capture the issue correctly.